src/utils/sortPeople.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable max-len */
+import { Person } from '../types';
+
+export const sortPeople = (
+  people: Person[],
+  order: string | null,
+  sort: string | null,
+) => {
+  let sortedPeople = [...people];
+
+  switch (sort) {
+    case ('name'):
+    case ('sex'):
+      if (order) {
+        sortedPeople = sortedPeople.sort((prevPerson, currentPerson) => currentPerson[sort].localeCompare(prevPerson[sort]));
+      } else if (!order) {
+        sortedPeople = sortedPeople.sort((prevPerson, currentPerson) => prevPerson[sort].localeCompare(currentPerson[sort]));
+      }
```

Comment:
You can use ternary inside sort method to avoid using `else if` and repeating yourself